src/ui/public/management/sections_register.js

```diff
@@ -34,19 +34,25 @@ management.register('data', {
 });
 
 management.register('elasticsearch', {
-  display: 'Elasticsearch',
+  display: i18n.translate('common.ui.management.elasticsearchDisplayName', {
```

Comment:
We do not need to translate it, here you can find all keywords and terms which should not be translated https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit